{PROJECT_NAME}/Configurations/Plists/GoogleService/Production/GoogleService-Info.plist

```diff
@@ -2,6 +2,33 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CLIENT_ID</key>
```

Comment:
This PR is for uploading staging builds to firebase, should we remove changes from `Production's` Google-Info.plist file?